docs/reuse_cluster.md

```diff
@@ -0,0 +1,72 @@
+## Reusing an already running cluster
+
+### Overview
+
+SCT allows running tests using a previously provisioned cluster. This helps in efficiently utilizing resources and reduces setup time for subsequent tests.
+
+Reusing a cluster is primarily intended for local development and debugging. Scenarios that can benefit from it include:
+- re-run a test without having to wait for a new cluster to be provisioned
+- re-run a test with updated (or different) configuration
+- update Scylla packages on an existing cluster and re-run the test
+
+### Prerequisites
+
+To provision a cluster for later reuse, the initial test run must be executed with the `post_behavior` configuration parameters set to `keep` value.
+
+This can be done in the test configuration YAML file as:
+```yaml
+post_behavior_db_nodes: "keep"
+post_behavior_loader_nodes: "keep"
+post_behavior_monitor_nodes: "keep"
+```
+For K8s cluster the configuration parameter to use is:
+```yaml
+post_behavior_k8s_cluster: "keep"
+```
+Alternatively, `SCT_POST_BEHAVIOR` environment variables can be set before running the test, for each node type to be kept:
+```bash
+export SCT_POST_BEHAVIOR_DB_NODES=keep
+export SCT_POST_BEHAVIOR_LOADER_NODES=keep
+export SCT_POST_BEHAVIOR_MONITOR_NODES=keep
+export SCT_POST_BEHAVIOR_K8S_CLUSTER=keep
```

Comment:
Lines 31 and 35 look as duplicates. Is it intentional?

Reply:
typo.. 
corrected